detox/src/android/AndroidExpect.test.js

```diff
@@ -280,6 +280,14 @@ describe('AndroidExpect', () => {
       it('should adjust slider to position', async () => {
         await e.element(e.by.id('sliderWithASimpleID')).adjustSliderToPosition(75);
       });
+
+      it('should perform accessibility actions (defaults to "activate")', async () => {
```

Comment:
Why did you implement Android and iOS differently? I see that iOS will throw on attempt to perform accessibility actions without parameters, while on Android you set a default value.

Maybe you should remove the default value?

Reply:
That's true, thanks. 
I've removed the Android default in https://github.com/wix/Detox/pull/3877/commits/7e6345759c46f8788ffa28ed8712b1f8aacf556d